examples/widget_examples.ipynb

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Replace by more generic docs, focussing on how to use the widgets. Avoid making anything complex or neutron specific.